src/ReplicatedStorage/Client/InstanceComponents/Misc/CursorLook.lua

```diff
@@ -62,6 +65,16 @@ function CursorLook.new(instance: Instance)
 	self._waistAngles = Vector3.new()
 	self._neckAngles = Vector3.new()
 
+	-- Get shared cursor property
+	local cursorProp = clientComm:GetProperty("Cursor")
+
+	-- Keep track of the mouse cursor
+	local mouseCursor
+	cursorProp:Observe(function(value)
```

Comment:
Using :Observe to keep a local variable tracked feels a little icky to me. Could you instead just poll the property when you use it? The performance should be roughly similar because RemoteProperty has a cache anyways (read: this won't cause network calls)